liota/dccs/iotcc.py

```diff
@@ -422,6 +432,7 @@ def store_device_info(self, uuid, name, dev_type, prop_dict):
 
     def write_entity_file(self, prop_dict, res_uuid):
         file_path = self.entity_file_path + '/' + res_uuid + '.json'
+        prop_dict.update({"Entity_Timestamp": datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S")})
```

Comment:
uptime library comes with a big warning (Source:https://pythonhosted.org/uptime/):

"Warning On most platforms, this module depends very heavily on ctypes. It has become painfully apparent that many less mainstream platforms ship with a broken version of this standard library module, either accidentally or deliberately. Please test your Python installation before using uptime."

Have you tested it on gateways like DK300 and Dell 5000?

Regarding datetime, it can actually be converted to epoch using:

(datetime.datetime(2012,04,01,0,0) - datetime.datetime(1970,1,1)).total_seconds()
Source: http://stackoverflow.com/questions/11743019/convert-python-datetime-to-epoch-with-strftime